### DISCUSSION TITLE
Add devcontainer "postStartCommand" support to Codespaces

### DISCUSSION THREAD

**Discussion by hediet** [community/community#41004] [Codespaces]
### Select Topic Area

Bug

### Body

The `postStartCommand` of devcontainer does not seem to be supported.

In my scenario, I'd like to configure the vscode [devcontainer.json](https://github.com/microsoft/vscode/blob/6aa9ea1b7bc428186a64b605f1d74092ab4b8ad9/.devcontainer/devcontainer.json) to start the watch background task, so I can run the tests once everything is setup.

However, the command I put in `postStartCommand` in my branch does not seem to be executed.

**Comment by joshaber** [comment]
Hi @hediet, Codespaces supports `postStartCommand`. I ran a quick test with the following devcontainer.json:
```jsonc
{
  "postStartCommand": "touch ~/poststart.txt"
}
```

And verified that the `~/poststart.txt` file existed after creating a codespace. You could also try running `>Codespaces: View Creation Log` in VS Code after creating the codespace to see if any errors are logged.

That said, judging from how you want to use the command, [`postAttachCommand`](https://containers.dev/implementors/json_reference/#lifecycle-scripts) might be a better fit. It will run in a VS Code terminal when VS Code attaches. That way you can see your watcher task and its output.
